package.json

```diff
@@ -78,12 +97,20 @@
     "underscore": "1.8.3",
     "underscore-mixins": "0.1.4"
   },
+  "nyc": {
+    "include": [
+      "src/"
+    ],
+    "sourceMap": false,
+    "instrument": false
+  },
   "config": {
     "commitizen": {
       "path": "node_modules/cz-conventional-changelog"
     },
     "ghooks": {
-      "pre-commit": "npm run test:ci && npm run check-coverage"
+      "commit-msg": "validate-commit-msg",
+      "pre-commit": "./scripts/lint-staged-files && npm run coverage && npm run check-coverage"
```

Comment:
It seems this part is not consistent at all with the boilerplate + other repos. I think `./scripts/lint-staged-files` should be enough